core/editor/loader/editor-loader-factory.php

```diff
@@ -38,14 +38,11 @@ public static function create() {
 	}
 
 	/**
-	 * If any of the V2 experiments is active, we should use the V2 loader.
+	 * If there are v2 packages enqueued, we should use the V2 loader.
 	 *
 	 * @return bool
 	 */
 	private static function should_use_v2_loader() {
-		$experiments_manager = Plugin::$instance->experiments;
-
-		return Collection::make( Editor_V2_Experiments::all() )
-			->some( fn ( $experiment ) => $experiments_manager->is_feature_active( $experiment ) );
+		return ! Editor_V2_Packages::collect()->is_empty();
```

Comment:
I think it should be based on the experiment and not based on the packages. The packages are being enqueued based on the experiments so just use the loader based on the same experiments.

Reply:
You can load packages without depending on the experiment, The idea is if third parties want to load some feature that uses v2 packages, it can be used, v1 still works it just load the infra of v2